app/build.gradle

```diff
@@ -109,6 +109,7 @@ dependencies {
       'androidx.work:work-runtime-ktx:2.4.0',
       'com.chaos.view:pinview:1.4.3',
       'com.github.bumptech.glide:glide:4.11.0',
+      'com.github.mmin18:realtimeblurview:1.2.1',
```

Comment:
As for Bazel, we also need to add a dependency to app module-level BUILD.bazel file.
https://github.com/oppia/oppia-android/blob/develop/app/BUILD.bazel